### PR TITLE
fix grammar

### PR DESCRIPTION
**Pull request by ShalokShalom**
Fixes articulation and grammar, checked by Grammarly.

### CODE REVIEW

File: docs/language-requests.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ time to time in the Highlight.js issue tracker and discussion group.
     This means that *there's no point in requesting a new language without also
     providing a 3rd party implementation* (we'll simply close "Please support
     language Xyz" issues with a link to this explanation). If you'd like to see
-    a particular language available but cannot implement it, the best way to
+    a particular language available that you cannot implement yourself, the best way to
```
Comment (joshgoebel) [inline docs/language-requests.rst:22]: perhaps using "unable" here could be better?

Comment (ShalokShalom) [reply docs/language-requests.rst:22]: Why?

File: docs/language-requests.rst [hunk 2]
```diff
@@ -7,8 +7,8 @@ time to time in the Highlight.js issue tracker and discussion group.
     Highlight.js does not have a fundamental plan for implementing new languages
     - i.e., the core team doesn't usually develop new languages. The core team
     instead focuses on parser development, bugs, and supporting the existing
-    languages. They also currently does not have time to review, merge and
-    maintain any additional languages (fixing bugs, dealing with issues, etc).
+    languages. They also do not have time to review, merge, and
+    maintain additional languages (fixing bugs, dealing with issues, etc).
```
Comment (joshgoebel) [inline docs/language-requests.rst:11]:
```suggestion
    languages. They also do not have time to review, merge, and maintain
    additional languages (fixing bugs, dealing with issues, etc).
```

**Comment by joshgoebel** [comment]
I like this.  Is grammarly happy with it?

**Comment by ShalokShalom** [comment]
Yep :)

**Pull request merged**